File: Arithmetic_OddNumbers.java
package arithmetic;

public class Arithmetic {

    public static void main(String[] args) {
        System.out.println("Welcome to our demonstration sequels");
        System.out.println("Hope you enjoy (and follow) the lessons.");
        System.out.println("");
        
        /* Use the Odd Number class. */
        OddNumbers odd_data = new OddNumbers(1, 100); // odd numbers between 1 and 100
        System.out.println(odd_data.prepResult());

        System.out.println();
    }

}

File: Arithmetic_OddNumbers.cs
using System;
using System.Collections.Generic;

namespace Arithmetic_CS
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Welcome to our demonstration sequels");
            Console.WriteLine("Hope you enjoy (and follow) the lessons.");
            Console.WriteLine("");

            /* Use the Even Number class. */
            int first = 1;
            int last = 100;
            List<int> answer;

            OddNumbers odd_data = new OddNumbers(first, last); // odd numbers between 1 and 100
            answer = odd_data.prepResult();

            Console.WriteLine("Odd numbers between " + first + " and " + last + " are:");
            Console.WriteLine(String.Join(", ", answer));

        }
    }
}
